### DISCUSSION TITLE
about the online test_x

### DISCUSSION THREAD

**Discussion by Legend94rz** [PaddlePaddle/PaddleSpatial#136] [BaiduKDDCup2022] [answered]
It seems like Turb1 has 15 days data, but Day1 only has 126 rows and Day15 only has 18 rows, the expected  #row of Day1 and Day15 are both 144(=24*6).

Should we handle these data missing ourselves? but the toy test_x and the training data still offer the line with all data missing, e.g.
```
TurbID,Day,Tmstamp,Wspd,Wdir,Etmp,Itmp,Ndir,Pab1,Pab2,Pab3,Prtv,Patv
1,1,00:00,,,,,,,,,,
```

**Comment by ramber1836** [comment, marked as answer]
You can handle the missing values as long as the outcome prediction has the right shape.
